README.md

```diff
@@ -58,7 +58,7 @@ In [infrastructure](infrastructure/), we present all the services which should b
 
 ## Laboratory Operator
 
-The Laboratory Operator (LabOperator) implements the backend logic necessary to spawn new laboratories starting from a predefined template. The instruction to install and configure the LabOperator are detailed [here](operators).
+The Laboratory Operator (LabOperator) implements the backend logic necessary to spawn new laboratories and persistent vm starting from a predefined template. The instruction to install and configure the LabOperator are detailed [here](operators).
```

Comment:
```suggestion
The Laboratory Operator (LabOperator) implements the backend logic necessary to spawn new laboratories and persistent VMs starting from a predefined template. The instruction to install and configure the LabOperator are detailed [here](operators).
```

Reply:
Thank you for the suggestion, web have updated the read.me in the last commit